apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/SubscriberClientTest.cs

```diff
@@ -149,16 +149,14 @@ public void Dispose()
             private class PullStream : StreamingPullStream
             {
                 public PullStream(TimeSpan writeAsyncPreDelay,
-                    IEnumerable<ServerAction> msgs, List<TimedId> extends, List<TimedId> acks, List<DateTime> writeCompletes,
+                    IEnumerable<ServerAction> msgs, List<DateTime> writeCompletes,
```

Comment:
So is this a list that we're passing in to collect values? Do we pass in a non-empty list? If not, could we create the list ourselves an expose it? (I suspect the order in which I'm reviewing the files isn't ideal.)